.travis.yml

```diff
@@ -67,7 +67,7 @@ before_install:
   - git remote add origin https://${github_api_token}@github.com/${UTPLSQL_REPO}
   # download latest utPLSQL release
   #- curl -LOk $(curl --silent https://api.github.com/repos/utPLSQL/utPLSQL/releases/latest | awk '/browser_download_url/ { print $2 }' | grep ".zip" | sed 's/"//g')
-  - git clone --depth=1 --branch=develop https://github.com/utPLSQL/utPLSQL.git $UTPLSQL_DIR
+  - git clone --depth=1 --branch=${SELFTESTING_BRANCH:-master} https://github.com/utPLSQL/utPLSQL.git $UTPLSQL_DIR
```

Comment:
We need to set `SELFTESTING_BRANCH=develop` in order for this change to work before we release 3.0.3 I think